pkg/security/probe/probe_ebpf.go

```diff
@@ -1060,6 +1060,9 @@ func (p *EBPFProbe) handleEvent(CPU int, data []byte) {
 			seclog.Errorf("failed to insert exec event: %s (pid %d, offset %d, len %d)", err, event.PIDContext.Pid, offset, len(data))
 			return
 		}
+
+		event.Exec.FileEvent.PathnameStr = p.fieldHandlers.ResolveFilePath(event, &event.Exec.FileEvent)
```

Comment:
Why do you to call this two function here, the path should have been resolved in the AddExecEntry ?